spec/lib/miq_automation_engine/miq_ae_discovery_spec.rb

```diff
@@ -4,12 +4,14 @@ module MiqAeDiscoverySpec
   include MiqAeEngine
   describe "MiqAeDiscovery" do
     before(:each) do
+      # admin user is needed to process Events
+      @admin = FactoryGirl.create(:user_with_group, :userid => "admin", :name => "Administrator")
```

Comment:
Does this really need to be called "admin"?

Do you want to use `User.seed` instead?